docs/user-guide.md

```diff
@@ -111,9 +138,9 @@ public class Strings {
 ```
 
 In this example, both methods are in the scope of `@NullMarked`, so plain `String`
-means "a reference to a `String` that can't be null". `@Nullable String` continues
-to mean "a reference to a `String` that can be null". Tools should warn you if you
-try to pass a "reference to a `String` that can be null" to `spaceIndex`, since its
+means "a reference to a string, not null". `@Nullable String` continues
+to mean "a reference to a string, or null". Tools should warn you if you
```

Comment:
Earlier https://github.com/jspecify/jspecify/pull/201/files#r662668740
Would making the distinction between the two kinds of calls explicit be useful?

Reply:
After the discussion in our last meeting, I've tried to make it so that we say as little as possible about code that is not `@NullMarked`. We dispensed with non-`@NullMarked` code above and I think we can leave it at that.